README.md

```diff
@@ -251,8 +279,17 @@ If you don't know where your `kubeconfig` is due to running against multiple clu
 oc whoami --loglevel=100
 # likely output will be $HOME/.kube/config 
 ``` 
+If you need to know information about your cluster:
 
-
+```bash
+# this will list all images, associated github repo, and the commit # currently running.
+# very useful to see if the image is running current code...or not.
```

Comment:
@jhadvig @zherman0 `oc adm release info --commits` is a pretty helpful nugget.  You need a fairly recent version of `oc`, but it will give you the `git` `commit` for whatever code is running for the `console-operator` (and all the operators).  From there, its pretty easy to search and see if things are current.